src/lib/components/Table/Table.js

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import naturalCompare from 'natural-compare-lite';
+import './Table.scss';
+
+import TableCell from './TableCell';
+import TableRow from './TableRow';
+
+class Table extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.generateData = this.generateData.bind(this);
+    this.generateHeader = this.generateHeader.bind(this);
+    this.generateRows = this.generateRows.bind(this);
+    this.getNewSortCondition = this.getNewSortCondition.bind(this);
+    this.sortData = this.sortData.bind(this);
+
+    this.state = {
+      columns: props.columns ? [...props.columns] : null,
+      data: props.data ? [...props.data] : null,
+      originalData: props.data ? [...props.data] : null,
+      sortCondition: props.sortCondition,
```

Comment:
Does the trailing , not get caught by linting?

Reply:
Current linting is set to always have a trailing comma (which I also don't like). I'll leave for now and we'll discuss our linting at dev catchup